modules/claims_api/app/workers/claims_api/updater_service.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+module ClaimsApi
+  class UpdaterService
+    extend ActiveSupport::Concern
+
+    def find_claim(claim_id)
```

Comment:
Why not put `claim = ClaimsApi::AutoEstablishedClaim.find(claim_id)` at line 14 instead/